src/main/java/ned/commands/AddEventCommand.java

```diff
@@ -0,0 +1,70 @@
+package ned.commands;
+
+import ned.Storage;
+import ned.TaskList;
+import ned.exceptions.NedException;
+
+import ned.tasks.Task;
+import ned.tasks.Event;
+
+import ned.Ui;
+
+import java.util.regex.Pattern;
```

Comment:
Can standardise the format of import statements, whether to have blank lines in between or not